modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+// Epic CNF-3901 (CNF-2133) (4.11), Story TELCODOCS-339
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="talo-backup-recovery_{context}"]
+= Recovering after a failed upgrade
+
+If an upgrade of a cluster fails, you can manually log in to the cluster and use the backup to return the cluster to its pre-upgrade state. There are two stages:
+
+Rollback:: If the attempted upgrade included a change to the platform OS deployment, it will be necessary to rollback to the previous version before running the recovery script.
+Recovery:: The recovery shuts down containers and uses files from the backup partition to relaunch containers and restore clusters.
+
+.Prerequisites
+
+* Install the {cgu-operator-first}.
+* Provision one or more managed clusters.
+* Install Advanced Cluster Management.
+* Log in as a user with `cluster-admin` privileges.
+* Run an upgrade that is configured for backup.
+
+.Procedure
+
+. Delete the the `ClusterGroupUpgrade` CR "clustergroupupgrades-group-du.yaml".
```

Comment:
Hei, i would not mention the "clustergroupupgrades-group-du.yaml". This yaml created the CR actually. therefore, i would say:
 
 Delete the `ClusterGroupUpgrade` CR that was created before

Reply:
OK